src/main/webapp/app/course/manage/course-management.component.ts

```diff
@@ -20,7 +20,6 @@ export class CourseManagementComponent implements OnInit, OnDestroy, AfterViewIn
     predicate: string;
     reverse: boolean;
     showOnlyActive = true;
-    showExamButton = false;
```

Comment:
I would like to keep this feature disabled on production for now. So we should keep this attribute.
You can set it to true for testing, but we should set it to false again before we deploy the new features to production